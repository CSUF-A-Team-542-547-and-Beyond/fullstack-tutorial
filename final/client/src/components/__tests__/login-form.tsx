import React from 'react';
import { mount, shallow, render } from 'enzyme';	//added for shallow rendering
//import { render, cleanup } from '../../test-utils';
import { cleanup } from '../../test-utils';
import LoginForm from '../login-form';

describe('Login Form', () => {
  // automatically unmount and cleanup DOM after the test is finished.
  afterEach(cleanup);

  it('renders without error', () => {
    //render(<LoginForm login={() => {}}/>);
    const wrapper = shallow(<LoginForm login={() => {}}/>);
  });
});
