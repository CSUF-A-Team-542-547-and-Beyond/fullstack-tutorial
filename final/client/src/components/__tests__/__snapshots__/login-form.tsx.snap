// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form login form matches the snapshot 1`] = `
<div
  className="css-12broor"
>
  <header
    className="css-1ujrq0t"
  >
    <svg
      className="css-cvyda1"
    >
      curve.svg
    </svg>
    <svg
      className="css-1qk13o0"
    >
      logo.svg
    </svg>
  </header>
  <svg
    className="css-h0j5i9"
  >
    rocket.svg
  </svg>
  <h1
    className="css-1r0stgl"
  >
    Space Explorer
  </h1>
  <form
    className="css-ccdq42"
    onSubmit={[Function]}
  >
    <input
      className="css-wotvke"
      data-testid="login-input"
      name="email"
      onChange={[Function]}
      placeholder="Email"
      required={true}
      type="email"
    />
    <button
      className="css-wwcn44"
      type="submit"
    >
      Log in
    </button>
  </form>
</div>
`;
